ds-caselaw-ingester/lambda_function.py

```diff
@@ -79,6 +89,8 @@ def send_retry_message(original_message: Dict[str, Union[str, int]], sqs_client:
             QueueUrl=os.getenv('SQS_QUEUE_URL'),
             MessageBody=json.dumps(retry_message)
         )
+    else:
+        print(f'Maximum number of retries reached for {original_message["consignment-reference"]}')
```

Comment:
Can we throw this up to Rollbar? Should be able to just raise an exception perhaps?